File: native-src/ios/ProduceItem.swift
import Foundation

struct ProduceItem: Identifiable, Codable {
    let id: String
    let name: String
    let category: String
    let currentPrice: Double
    let level: String
}

// 模擬歷史資料用的結構
struct PriceHistory: Identifiable {
    let id = UUID()
    let date: String
    let price: Double
}

File: native-src/android/ProduceItem.java
package com.example.produceapp;

public class ProduceItem {
    public String id;
    public String name;
    public String category;
    public double currentPrice;
    public String level;

    public ProduceItem(String id, String name, String category, double currentPrice, String level) {
        this.id = id;
        this.name = name;
        this.category = category;
        this.currentPrice = currentPrice;
        this.level = level;
    }
}
